app/components/forms/session-speaker-form.js

```diff
@@ -467,6 +468,10 @@ export default Component.extend(FormMixin, {
     return orderBy(countries, 'name');
   }),
 
+  languages: computed(function() {
```

Comment:
I don't think we need this to be a computed property. Both `languages` and `countries` can just be be direct properties on the object.

```js
countries: orderBy(countries, 'name'),
languages: orderBy(languages, 'name')
```